OracleDatabase/RAC/OracleRealApplicationClusters/DOCKER_README.md

```diff
@@ -0,0 +1,669 @@
+
+CMAN_HOSTNAME=###Connection Manager Host Name###
+
+CMAN_IP=###Connection manager Host IP###
+
+ASM_DISCOVERY_DIR=####ASM disk location insdie the container. By default it is /dev######
+
+COMMON_OS_PWD_FILE=###You need to pass the file name to setup grid and oracle user password. If you specify ORACLE_PWD_FILE, GRID_PWD_FILE and DB_PWD_FILE then you do not need to specify this env variable###
+
+ORACLE_PWD_FILE=###You need to pass the file name to set the password for oracle user.###
+
+GRID_PWD_FILE=###You need to pass the file name to set the password for grid user.###
+
+DB_PWD_FILE=###You need to pass the file name to set the password for DB user i.e. sys.###
+
+REMOVE_OS_PWD_FILES=###You need to set this to true to remove pwd key file and password file after resetting password.###
+```
+
+## Sample Container Files for Older Releases
+This project offers sample container files for Oracle Grid Infrastructure and Oracle Real Application Clusters for dev and test:
+  
+ * Oracle Database 19c Oracle Grid Infrastructure (19.3) for Linux x86-64
+ * Oracle Database 19c (19.3) for Linux x86-64
+ * Oracle Database 18c Oracle Grid Infrastructure (18.3) for Linux x86-64
+ * Oracle Database 18c (18.3) for Linux x86-64
+ * Oracle Database 12c Release 2 Oracle Grid Infrastructure (12.2.0.1.0) for Linux x86-64
+ * Oracle Database 12c Release 2 (12.2.0.1.0) Enterprise Edition for Linux x86-64
+  
+ **Notes:** 
+ * Note that the Oracle RAC on Docker Container releases are supported only for test and development environments, but not for production environments.
+ * If you are planning to build and deploy Oracle RAC 18.3.0, you need to download Oracle 18.3.0 Grid Infrastructure and Oracle Database 18.3.0 Database. You also need to download Patch# p28322130_183000OCWRU_Linux-x86-64.zip from [Oracle Technology Network](https://www.oracle.com/technetwork/database/database-technologies/clusterware/downloads/docker-4418413.html). Stage it under dockerfiles/18.3.0 folder.
+ * If you are planning to build and deploy Oracle RAC 12.2.0.1, you need to download Oracle 12.2.0.1 Grid Infrastructure and Oracle Database 12.2.0.1 Database. You also need to download Patch# p27383741_122010_Linux-x86-64.zip from [Oracle Technology Network](https://www.oracle.com/technetwork/database/database-technologies/clusterware/downloads/docker-4418413.html). Stage it under dockerfiles/12.2.0.1 folder.
+  
+## Section 9 : Support
+
+At the time of this release, Oracle RAC is supported for Oracle Linux 7. To see current Linux support certifications, see:
+
+## Section 10 : License
+
+To download and run Oracle Grid and Database, regardless of whether inside or outside a container, you must download the binaries from the Oracle website and accept the license indicated on that page.
+
+All scripts and files hosted in this project and GitHub docker-images/OracleDatabase repository required to build the container  images are unless otherwise noted, released under UPL 1.0 license.
```

Comment:
```suggestion
All scripts and files hosted in this repository which are required to build the container  images are, unless otherwise noted, released under UPL 1.0 license.
```